composer.json

```diff
@@ -34,6 +34,11 @@
     "conflict": {
         "fzaninotto/faker": "*"
     },
+    "suggest": {
+        "ext-dom": "(\\Faker\\Provider\\HtmlLorem) Required for generating random HTML.",
+        "ext-iconv": "(\\Faker\\Provider\\ru_RU\\Text::realText()) Required for generating real Russian text.",
```

Comment:
Let’s use Symfony Polyfill instead.

Reply:
Couldn't find a Symfony Polyfill for `ext-dom`. So should we keep `ext-dom` in `suggest` and replace `ext-iconv` with `symfony/polyfill-iconv` (but in `required`)?

Reply:
Yes, I meant only the iconv 😉